website/docs/d/vapp_vm.html.markdown

```diff
@@ -71,6 +71,16 @@ The following arguments are supported:
 * `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
```

Comment:
**Not** introduced by this PR, but maybe you could fix this sample output formatting on the way:

![Screenshot 2020-02-17 at 13 21 03](https://user-images.githubusercontent.com/11330689/74649546-7810f200-5188-11ea-8e85-e94889f21121.png)

CC: @dataclouder (in case you planned/fixed this too)

Reply:
Fixed. 
<img width="962" alt="image" src="https://user-images.githubusercontent.com/15804230/74722882-583e0480-5242-11ea-9ee4-e28a49342bb1.png">